### PR TITLE
conan example added

### PR DESCRIPTION
**Pull request by mbaykara**
example adopted with Conan based on CMake

### CODE REVIEW

**Review by prince-chrismc — COMMENTED** [review]
How would you feel about only using one cmake file? I think that's better for show how it works... you script as a few extras I was not expecting.

My personal preference is for the cmake_find_package[_multi] generators. I think it's nice to show the similarities.

**Comment by mbaykara** [comment]
I've changed accordingly and removed the second CMake file

File: conan-example/CMakeLists.txt [hunk 1]
```diff
@@ -1,15 +1,7 @@
 cmake_minimum_required(VERSION 3.16)
```
Comment (prince-chrismc) [inline conan-example/CMakeLists.txt:1]:
```suggestion
cmake_minimum_required(VERSION 3.8)
```

**Review by prince-chrismc — CHANGES_REQUESTED** [review]
Super close!

**Comment by mbaykara** [comment]
it should be done now

**Review by prince-chrismc — APPROVED** [review]

**Pull request merged**